.devcontainer/Dockerfile

```diff
@@ -0,0 +1,22 @@
+ARG BCI_IMAGE=registry.suse.com/bci/bci-base:latest
+FROM ${BCI_IMAGE} as bci
+FROM bci
+RUN zypper update -y && \
+    zypper && \
+    zypper install -y   \
+        ca-certificates \
+        strongswan      \ 
+        git             \
+        go              \
```

Comment:
Let's install Go via curl and tar. We shouldn't be beholden to the package manager. Let's use the latest version.